libretro-play/add-riscv64-build.patch

```diff
@@ -11,6 +11,17 @@ index dfb0ce09..b3e77d02 100644
  	set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -msse -msse2 -mssse3")
  endif()
  
+diff --git a/Source/ee/IPU.h b/Source/ee/IPU.h
+index 2c860a19..a46cb5c8 100644
+--- a/Source/ee/IPU.h
++++ b/Source/ee/IPU.h
+@@ -1,5 +1,6 @@
+ #pragma once
+ 
++#include <algorithm>
```

Comment:
Could you please try to upstream this?

Reply:
https://github.com/jpd002/Play-/pull/1389